modules/standard/FileSystem.chpl

```diff
@@ -21,12 +21,12 @@
 
    The FileSystem module focuses on file and directory properties and
    operations.  It does not cover every interaction involving a file -
-   for instance, path specific operations live in the :mod:`Path` module, while
-   opening, writing to, or reading from a file appropriately live in the
+   for instance, path-specific operations live in the :mod:`Path` module, while
+   opening, writing to, or reading from a file live in the
    :mod:`IO` module.  Rather, it covers cases where the user would prefer a file
-   or directory handled wholesale and/or with minimal interaction.  The
-   :proc:`copy` functionality belongs here, as does :proc:`chmod`,
-   :proc:`mkdir`, and :proc:`remove`, for instance.  Also included are
+   or directory handled wholesale and/or with minimal interaction.  For example,
+   this module contains file :proc:`copy` functionality, as well as :proc:`chmod`,
```

Comment:
I don't really like "this module contains the file copy functionality".  What I was hoping to do was build on the previous statement of "wholesale" operations by saying "it contains file copy functionality, as well as ...."  Alternatively, I could imagine saying simply "it contains copy functionality", though then it seems vague as to what's being copied (a file?  a directory?  a string?).  What do you think?